pkg/util/helper/predicate.go

```diff
@@ -34,16 +34,9 @@ import (
 // - execution controller working in karmada-controller-manager
 // - work status controller working in karmada-controller-manager
 func NewExecutionPredicate(mgr controllerruntime.Manager) predicate.Funcs {
-	predFunc := func(eventType string, object client.Object) bool {
```

Comment:
Can we remove this `eventType` parameter, as it doesn't care about the event type anymore?